apps/docs/src/development/theming.md

```diff
@@ -68,3 +68,17 @@ const { store: colorScheme } = useColorMode();
 ```
 
 Alternatively, you can use the [OnyxColorSchemeDialog](https://storybook.onyx.schwarz/?path=/docs/support-colorschemedialog--docs) component to build your own custom component.
+
+### Transistion
```

Comment:
Please also create a changset for the new ``useThemeTransistion` composable + link the docs page :)